.circleci/config.yml

```diff
@@ -13,15 +13,16 @@ jobs:
       - run: npm run lint
   lint-client:
     docker:
-      - image: circleci/node
+      - image: circleci/node:10.19
```

Comment:
Fix node version at v10.19
CircleCI pipelines were failing for newer node versions.